### PR TITLE
Updated getting-started.md

### PR DESCRIPTION
**Pull request by davidshq**
## Description
Improvements to the getting-started.md documentation. Specifically:
1. Now explicitly note that when node is installed one must run the scripts from within the local gutenberg repo directory.
2. Broke into additional sections Local Environment.
   - Added Step 1: Installing a Local Environment
   - Changed "Using Docker" to "Quickest Method: Using Docker"
   - Prepended "Alternative Method:" to "Using an Existing Local WordPress Install"
   - Added Step 2: Accessing and Configuring the Local WordPress Install
3. Fleshed out the commands needed to add environment variables (especially on Windows, export will not work).
4. Changed verbiage regarding shutting down/starting up docker instance.
5. Clarified that the Toggling Debug Systems command will only work on Linux/MacOS (I do not have the correct command for Windows yet).

### CODE REVIEW

File: docs/contributors/getting-started.md [hunk 1]
```diff
@@ -17,9 +17,10 @@ This will build Gutenberg, ready to be used as a WordPress plugin!
 
 If you don't have a local WordPress environment to load Gutenberg in, we can help get that up and running, too.
 
-## Local Environment
+## Local Environment
```
Comment (ZebulanStanphill) [inline docs/contributors/getting-started.md]: There's an unnecessary space added here.

File: docs/contributors/getting-started.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Gutenberg is a Node.js-based project, built primarily in JavaScript.
 
 The first step is to install a recent version of Node. The easiest way (on MacOS, Linux, or Windows 10 with the Linux Subsystem) is by installing and running [nvm]. Once `nvm` is installed, you can install the correct version of Node by running `nvm install` in the Gutenberg directory.
 
-Once you have Node installed, run these scripts:
+Once you have Node installed, run these scripts from within your local gutenberg repository:
```
Comment (ZebulanStanphill) [inline docs/contributors/getting-started.md]: I think "gutenberg" should be capitalized.

File: docs/contributors/getting-started.md [hunk 3]
```diff
@@ -46,22 +47,32 @@ In Windows, you can set the `WP_DEVELOP_DIR` environment variable using the appr
...
-### Toggling Debug Systems
+If you're running [e2e tests](/docs/contributors/testing-overview.md#end-to-end-testing), this change will be used correctly.
+
+To shut down this local WordPress instance run `npm run env stop`. To start it back up again, run `npm run env start`.
+
+#### Toggling Debug Systems
 
 WordPress comes with specific [debug systems](https://wordpress.org/support/article/debugging-in-wordpress/) designed to simplify the process as well as standardize code across core, plugins and themes. It is possible to use environment variables (`LOCAL_WP_DEBUG` and `LOCAL_SCRIPT_DEBUG`) to update a site's configuration constants located in `wp-config.php` file. Both flags can be disabled at any time by running the following command:
+
+Example on Linux/MacOS:
```
Comment (ZebulanStanphill) [inline docs/contributors/getting-started.md]: "MacOS" should be written as "macOS".

**Review by ZebulanStanphill — CHANGES_REQUESTED** [review]
I see a few minor typos, but otherwise this looks like a good improvement to me.

File: docs/contributors/getting-started.md (hunk 1 above)
Comment (davidshq) [reply docs/contributors/getting-started.md]: Removed

File: docs/contributors/getting-started.md (hunk 2 above)
Comment (davidshq) [reply docs/contributors/getting-started.md]: Done.

File: docs/contributors/getting-started.md (hunk 3 above)
Comment (davidshq) [reply docs/contributors/getting-started.md]: Done.

**Review by ZebulanStanphill — APPROVED** [review]

**Review by Soean — APPROVED** [review]

**Comment by Soean** [comment]
Looks good, thanks for improving the docs.

**Pull request merged**